lib/adzerk/instant_count.rb

```diff
@@ -11,5 +11,10 @@ def bulk(data={})
       response = @client.post_json_request(url, data)
       parse_response(response)
     end
+
+    def instant_counts()
+      url = "instantcounts/network?days=7"
```

Comment:
API documentation don't show that days is required. Is there a reason it's added here?

Reply:
I wonder if the docs are out of date? I got this response when trying to send the request without the days:
```
Adzerk::ApiError:
       cannot request totals for a network. please specify a time period within the last 90 days.
```

Reply:
Either way, I realize that it is not formatted correctly and will updating depending on the result

Reply:
Updated